src/http/client.rs

```diff
@@ -430,8 +430,9 @@ impl Http {
     ///
     /// Refer to Discord's [docs] for field information.
     ///
-    /// **note:** Creating a command with the same name as an existing command for your application
-    /// will overwrite the old command.
+    /// **Note:**
```

Comment:
Colon should appear after the asterisks to be consistent with other `Note` headings.
```suggestion
    /// **Note**:
```